cg/meta/upload/scout/hk_tags.py

```diff
@@ -6,10 +6,16 @@
 
 
 class CaseTags(BaseModel):
-    snv_vcf: Set[str] = Field(..., description="vcf_snv for rare disease and vcf_cancer for cancer")
+    snv_vcf: Set[str] = Field(
+        None, description="vcf_snv for rare disease and vcf_cancer for cancer"
+    )
+    # snv_vcf: Set[str] = Field(..., description="vcf_snv for rare disease and vcf_cancer for cancer")
     snv_research_vcf: Set[str] = Field(None, description="vcf_snv_research for rare disease")
+    # sv_vcf: Set[str] = Field(
+    #     ..., description="vcf_cancer_sv for rare disease and vcf_sv_cancer for cancer"
+    # )
```

Comment:
I left this commented out since I wasn't sure it would break something as I'm not super following this "..." syntax. I will have another look before deleting these just in case.

Reply:
Yes, you have to check that this syntax is correct https://docs.pydantic.dev/usage/models/#required-fields.

Are `snv_research_vcf`, `snv_research_vcf` still required fields that accept `None`?

> If you want to specify a field that can take a None value while still being required, you can use Optional with ...
> c: Optional[int] = Field(...)